content/en/tutorial/10-signals.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Signals
+prev: /tutorial/09-error-handling
+next: /tutorial/11-links
+solvable: true
+---
+
+# Signals
+
+In chapter four, we saw how Preact manages state with the help of
+**hooks**. Hooks are powerful, but they always re-render the whole component whenever the state changes.
+
+**Signals** is a other way to manage state in Preact. Unlike hooks, which re-render the whole component whenever the state changes, the signals update only the
+parts of the UI that actually uses them. A signal remembers its value and automatically tracks where it is used, so when the value
+changes, only the parts that depend on it will update.
+
+### Creating a signal
```

Comment:
I feel like a lot of this content should be copy/pasted from the Signals guide page or intro blog. It looks like you wrote this entirely yourself, which is impressive, but the language and examples diverge quite a bit despite containing similar information.

Don't have time to review more right now, but I can take a look at this myself at some point.